app/src/main/java/com/pokerio/app/utils/GameState.kt

```diff
@@ -304,7 +305,9 @@ object GameState {
         playerJoinedCallbacks.forEach { it.value(player) }
     }
 
+    @Throws(IllegalArgumentException::class)
     fun removePlayer(playerHash: String, newAdmin: String? = null) {
+        println("Player to remove: $playerHash")
```

Comment:
Was this supposed to be here?

Reply:
I SINCERELY APOLOGIZE for my negligence.